snake.go

```diff
@@ -29,6 +29,11 @@ const (
 	HEIGHTBLOCKS = 13
 )
 
+type Point struct {
```

Comment:
For history of this structure, it could be nice to have a comment explaining why we are not using image.Point using the comment from @conejoninja.

Reply:
I will add the comment.